### ISSUE TITLE
implementing prior bounding boxes as keras layers?

### ISSUE THREAD

**Issue by BenMacKenzie** [rykov8/ssd_keras#71]
is there any particular reason to implement the bounding box priors as a layer in the keras model?  if i'm reading this correctly, it only ever gets used again during post-processing of the prediction.  is it to 'auto-configure' the priors based on input size etc... by detecting input from previous layer?

j

**Comment by rykov8** [comment]
@BenMacKenzie yep, it's for auto-configuring priors based on a layers' outputs.

**Comment by SeaOfOcean** [comment]
Make PriorBox a keras layer, will it compute input gradient in backward?
